README.md

```diff
@@ -22,19 +22,15 @@ For more information, please check our [documentation on colin.readthedocs.io](h
 
 ## Usage
 
-### How to test colin locally
+### How to test Colin locally
```

Comment:
I assume that we are not testing Colin, so this should read "How to Test a Container Image with Colin"

Reply:
It was probably meant as test=try..

I suggest not to use `Container Image` alone, since colin can check also dockerfiles and containers. Maybe `s/test/try`? Or container/image/dockerfile.

It will be good to correct the target also in the Makefile/Readme. (e.g. `TEST_IMAGE_NAME` -> `TARGET_NAME`,...)